### PR TITLE
Updated internship listings

### PR DESCRIPTION
**Pull request by sarahcrack**
### Trello card

https://trello.com/c/S1WGwTYm/6685-upload-latest-internships-listings

### Context

The internships team have some updated listing details that they would like to be uploaded.

### Changes proposed in this pull request

Updated CSV file and ran rake task.

### CODE REVIEW

File: app/views/content/train-to-be-a-teacher/teaching-internships.md [hunk 1]
```diff
@@ -10,23 +10,14 @@ promo_content:
   - content/train-to-be-a-teacher/promos/eta-promo-internships
 backlink: /
 fullwidth: true
-navigation: 20.30
+navigation: 20.25
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/teaching-internships.md:13]: @sarahcrack @martyn-w I think the script needs updating slightly as this change isn't needed

Emma changed this nav order from `20.25` to `20.30` when she created the new placements page as part of https://github.com/DFE-Digital/get-into-teaching-app/pull/4213 so we need to keep it at `20.30`

Comment (gemmadallmandfe) [reply app/views/content/train-to-be-a-teacher/teaching-internships.md:13]: (completely forgot about the internship script when approving Emma's PR!)

**Review by martyn-w — APPROVED** [review]
👍  LGTM

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**